lib/gringotts/gateways/trexle.ex

```diff
@@ -50,26 +50,33 @@ defmodule Gringotts.Gateways.Trexle do
   ```
   iex> amount = 100
 
-  iex> card = %{
-    name: "John Doe",
+  iex> card = %CreditCard{
     number: "5200828282828210",
-    expiry_month: 1,
-    expiry_year: 2018,
-    cvc: "123",
-    address_line1: "456 My Street",
-    address_city: "Ottawa",
-    address_postcode: "K1C2N6",
-    address_state: "ON",
-    address_country: "CA"
+    month: 12,
+    year: 2018,
+    first_name: "John",
+    last_name: "Doe",
+    verification_code: "123",
+    brand: "visa"
   }
 
```

Comment:
@jyotigautam  you are assigning a value for card whereas you are declaring module attribute for address, since these examples are going to be executed in `iex`, module attribute will not work. change it to below throughout the examples.

```
address = %Address{
 +    street1: "123 Main",
 +    street2: "Suite 100",
 +    city: "New York",
 +    region: "NY",
 +    country: "US",
 +    postal_code: "11111",
 +    phone: "(555)555-5555"
 +  }
```

Reply:
Have assigned this address struct to a variable instead of using it as a module attribute as you said.